aws/terraform/infra/s3.tf

```diff
@@ -0,0 +1,6 @@
+resource "aws_s3_bucket" "data_sets" {
```

Comment:
do we need both this bucket and the one created in the datastore module?

Reply:
That's a really good question, which I believe we should strive for answering in the documentation somewhere as well. To answer your question straight on, no, we do not need both these buckets. The only required bucket is the one provisioned by the datastore module. 

However, this bucket you tagged above, is incredibly convenient for ingress/egress data to Metaflow. If you had some external service that you wanted to inspect the results of some Metaflow job one could obviously use the Metadata API to query for it, but another approach would be having a separate S3 bucket where Metaflow could write to. This approach also works for loading in datasets from an external source if they're dumped here.

This was done to ensure that we were not reading/writing to the S3 bucket that Metaflow owned to pretty much avoid any confusion and possible human error. 

Does that make sense?